File: PHP/1-Foundation/5-Calculator-Again/code.php
<?php

do {
   echo 'x: ';
   $x = (int) fgets(STDIN);

   echo 'y: ';
   $y = (int) fgets(STDIN);

   echo 'Operation: ';
   $op = rtrim(fgets(STDIN));

   // Blank line.
   echo "\n";

   if ($op == 'a') {
      $op_sym = '+';
      $result = $x + $y;
   }
   elseif ($op == 's') {
      $op_sym = '-';
      $result = $x - $y;
   }
   elseif ($op == 'm') {
      $op_sym = '*';
      $result = $x * $y;
   }
   elseif ($op == 'd') {
      $op_sym = '/';
      $result = $x / $y;
   }
   elseif ($op == 'r') {
      $op_sym = '%';
      $result = $x % $y;
   }
   elseif ($op == 'e') {
      $op_sym = '**';
      $result = $x ** $y;
   }

   if (isset($op_sym))
      echo $x, ' ', $op_sym, ' ', $y, ' = ', $result;
   else
      echo 'Unknown operation.';


   // Restart prompt.
   echo "\n\nRestart? ('y' for yes, 'n' for no): ";
   $restart = rtrim(fgets(STDIN));

   // Blank line before next computation.
   echo "\n";
}

while ($restart == 'y');

File: Python/01-Foundation/4-Rudimentary-Calculator/code.py
x = int(input('x: '))
y = int(input('y: '))
op = input('Operation: ')

# Blank line before output.
print()

if op == 'a':
   print(x, '+', y, '=', x + y)
elif op == 's':
   print(x, '-', y, '=', x - y)
elif op == 'm':
   print(x, '*', y, '=', x * y)
elif op == 'd':
   print(x, '/', y, '=', x / y)
elif op == 'e':
   print(x, '**', y, '=', x ** y)
elif op == 'r':
   print(x, '%', y, '=', x % y)
else:
   print('Unknown operation.');